### PR TITLE
Fix 'doppler configs delete' and 'doppler environments delete' commands

### PR DESCRIPTION
**Pull request by nmanoogian**
...by passing query params to API function.

This fixes a regression introduced in bd094d1e7023174c24e58fbf1647ebe87bebce95

Closes ENG-5849

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -38,9 +38,14 @@ builds:
...
-    darwin: macOS
-    386: i386
+  rlcp: true
+  name_template: >-
+    {{ .ProjectName }}_
+    {{- if eq .Os "darwin" }}macOS
+    {{- else }}{{ .Os }}{{ end }}_
+    {{- if eq .Arch "386" }}i386
+    {{- else }}{{ .Arch }}{{ end }}
+    {{- if .Arm }}v{{ .Arm }}{{ end }}
```
Comment (Piccirello) [inline .goreleaser.yml]: Will this `Arm` line change the existing naming scheme?

**Review by Piccirello — COMMENTED** [review]
🤦‍♂️

File: .goreleaser.yml (hunk 1 above)
Comment (nmanoogian) [reply .goreleaser.yml]: Good callout, I meant to add a comment that I need to test this. The goreleaser docs said that these templates would be equivalent but I just did some testing locally with `goreleaser release --snapshot` and I'm getting different artifacts before and after this change. I'll need to take a closer look at this

Comment (nmanoogian) [reply .goreleaser.yml]: I force pushed a change and I think I've got it!

I pulled the defaults from https://goreleaser.com/customization/nfpm/?h=nfpms and https://goreleaser.com/customization/archive/?h=archives to make the necessary modifications.

I ran `rm -rf dist && goreleaser release --snapshot; (find dist | sort)` to get the generated list of artifacts and they look identical apart from the version hashes 👍 

Screenshot 2023-02-16 at 5 19 50 PM

Comment (Piccirello) [reply .goreleaser.yml]: Nice verification! I much preferred goreleaser's old method for defining this, but so is life...

File: pkg/http/api.go [hunk 2]
```diff
@@ -685,7 +685,7 @@ func DeleteEnvironment(host string, verifyTLS bool, apiKey string, project strin
 	params = append(params, queryParam{Key: "project", Value: project})
 	params = append(params, queryParam{Key: "environment", Value: environment})
 
-	url, err := generateURL(host, "/v3/environments/environment", nil)
+	url, err := generateURL(host, "/v3/environments/environment", params)
```
Comment (Piccirello) [inline pkg/http/api.go:688]: I manually re-reviewed my PR that broke these and couldn't find any other instances

**Review by Piccirello — APPROVED** [review]

File: pkg/http/api.go (hunk 2 above)
Comment (nmanoogian) [reply pkg/http/api.go:688]: Cool, same here -- I found `configs` by testing and during re-review I found `environments`

File: .goreleaser.yml (hunk 1 above)
Comment (nmanoogian) [reply .goreleaser.yml]: Agreed 😞

**Pull request merged**